rai_utils/setup.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+import setuptools
+
+with open('rai_utils/version.py') as f:
+    code = compile(f.read(), f.name, 'exec')
+    exec(code)
+
+# Fetch ReadMe
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+# Use requirements.txt to set the install_requires
+with open('requirements.txt') as f:
+    install_requires = [line.strip() for line in f]
+
+setuptools.setup(
+    name=name,  # noqa: F821
+    version=version,  # noqa: F821
+    author="Roman Lutz, Ilya Matiach, Ke Xu",
```

Comment:
Very generous of you 🤣 I guess it's fine in any way you like it, you can definitely remove my name, too.